src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,16 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val user = User()
+    user.initUser()
+    return user
 }
 
 internal fun User.address(initAddress: Address.() -> Unit): User {
-    TODO("Implement me!!!")
+    val address = Address()
```

Comment:
Same as in an exemple above, we can use scoped functions. 
```suggestion
   return apply { address = Address().apply(initAddress) }
```